Sources/Geocodable.swift

```diff
@@ -22,52 +22,81 @@ public protocol Geocodable {
     func geocodeLocality(_ string: String,
                          forCountry country: String,
                          completion: @escaping (Result<GeocodedAddress, GeocodingError>) -> ())
-
-}
-
-public extension Geocodable {
-
-    /// Geocodes the given string into a geocoded address.
+    
+    /// Geocodes the given string into a geocoded location.
     ///
     /// - Parameters:
     ///   - string: The string to geocode.
     ///   - country: The country to use as a filter. Default value is "US".
     ///   - completion: The completion block.
     func geocodeLocality(_ string: String,
```

Comment:
I created a new function mainly due to compatibility reasons. I didn't want to break existing projects that might already be using this library. If you think it makes more sense as a single function, I can update it.